.coderabbit.yaml

```
@@ -0,0 +1,106 @@
+code_generation:
+  language: "en"
+  # Exclude API folder from code generation and reviews
+  exclude_patterns:
+    - "api/**/*"
+  path_instructions:
+    "frontend/src/app/**/*.{ts,tsx,js,jsx}": |
+      Single-line comment for route/page purpose.
+      Document key data fetching or state management.
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)